zap/generate/configurator.go

```diff
@@ -47,6 +47,8 @@ func (tg *TemplateGenerator) renderZapTemplate(configurator *zap.Configurator, x
 	ce := x.SelectElement("configurator")
 	if ce == nil {
 		ce = x.CreateElement("configurator")
+		ce.CreateAttr("xmlns:xsi", "http://www.w3.org/2001/XMLSchema-instance")
+		ce.CreateAttr("xsi:noNamespaceSchemaLocation", "../../zcl.xsd")
 	}
```

Comment:
```suggestion
		ce = x.CreateElement("configurator")
	}
	ce.CreateAttr("xmlns:xsi", "http://www.w3.org/2001/XMLSchema-instance")
	ce.CreateAttr("xsi:noNamespaceSchemaLocation", "../../zcl.xsd")
```
I'm guessing you want this added to existing templates as well, but if it's only meant for new templates, ignore this suggestion.

Reply:
Thank you! I wasn't sure about mass-updating existing ones, even though it is ideal (so all of them properly ref schema), but realized later that re-formatting of generated file seems to be quite common in the matter repo 😅🤣